openvpn/protonvpn/update-port.sh

```diff
@@ -5,6 +5,10 @@
 
 TRANSMISSION_PASSWD_FILE=/config/transmission-credentials.txt
 
+transmission_username=$(head -1 ${TRANSMISSION_PASSWD_FILE})
+transmission_passwd=$(tail -1 ${TRANSMISSION_PASSWD_FILE})
+transmission_settings_file=${TRANSMISSION_HOME}/settings.json
```

Comment:
What if the file does not exist/is empty?

Reply:
Seems like an unlikely case since the transmission-credentials.txt file is automatically created/updated by transmission-openvpn. The file is populated with the TRANSMISSION_RPC_USERNAME and TRANSMISSION_RPC_PASSWORD environment variables. These variables are set as empty in the [Dockerfile](https://github.com/haugene/docker-transmission-openvpn/blob/ad0d9972d1fd14b33e3949bd0878f9c0ba02477c/Dockerfile#L99C3-L99C3) and [over-ridden in openvpn/start.sh](https://github.com/haugene/docker-transmission-openvpn/blob/ad0d9972d1fd14b33e3949bd0878f9c0ba02477c/openvpn/start.sh#L225) if supplied by in the docker compose file (or from secrets). 

If the transmission-credentials file is manually modified or deleted after the container has been created then it will be automatically modified/re-created as per above when the container is restarted.